### PR TITLE
removed django 1.3 check from test

### PR DESCRIPTION
**Pull request by auvipy**
new separate PR

### CODE REVIEW

File: tests/tests.py [hunk 1]
```diff
@@ -44,16 +44,7 @@ def assert_tags_equal(self, qs, tags, sort=True, attr="name"):
...
-            form_str %= {
-                "help_start": '<span class="helptext">',
-                "help_stop": "</span>"
-            }
-        else:
-            form_str %= {
-                "help_start": "",
-                "help_stop": ""
-            }
+        form_str %= {"help_start": "", "help_stop": ""}
```
Comment (frewsxcv) [inline tests/tests.py]: This is incorrect. This should be the *first* branch of the conditional (`if django.VERSION >= (1, 3):`).

**Comment by auvipy** [comment]
oops seems removed wrong part. updating!!!

**Comment by auvipy** [comment]
updated

**Comment by frewsxcv** [comment]
Thanks @auvipy!

**Pull request merged**